src/three-components/shader-chunk/common.glsl.js

```diff
@@ -9,19 +9,24 @@ import {texelConversions} from './encodings_pars_framgment.glsl.js'
 export const getDirectionChunk = /* glsl */ `
 vec3 getDirection(vec2 uv, int face) {
     uv = 2.0 * uv - 1.0;
-    vec3 direction;
+    vec3 direction = vec3(clamp(uv, -1.0, 1.0), 1.0);
+    uv = abs(uv);
+    float over = max(uv.x, uv.y) - 1.0;
+    if(over > 0.0) direction.z -= over;
```

Comment:
This is an interesting tweak. What case is being handled here?

Reply:
Added a comment above.

Reply:
💯